build_tools/gradle/build_tflite_android_library.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Cross-compile IREE's TFLite Java Bindings with Gradle and CMake. Produces an
```

Comment:
Are these instructions still accurate? https://github.com/google/iree/tree/main/bindings/tflite/java#building-the-library

Reply:
Yes (so far as I can tell). Manually building the bindings with just Gradle still works for me and others. Its github actions that that were the challenge, causing this extra work around. 

Ideally, users should still primarily rely on Gradle, since it will handle appropriating the correct dependencies (where as I'm specifying a specific set of dependencies in the Docker image).